app/models/organizations/judge_team.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class JudgeTeam < Organization
+  scope :available_for_priority_case_distribution, -> { where(automated_priority_case_distribution: true) }
```

Comment:
Read this suggestion last.
```suggestion
  scope :pushed_priority_cases_allowed, -> { where(accepts_pushed_priority_cases: true) }
```